CHANGES_NEXT_RELEASE

```diff
@@ -7,6 +8,7 @@
 - Unit tests are executed as part of Travis CI.
 - Fixing issues in the provisioning API (#99 and #100) as result of npm-shrinkwrap file update.
 - Fix: active attributes are not updated in CB for preregistered devices (#107).
+- Fix: update registration requests coming from devices are not correctly hanlded (#109).
```

Comment:
Duplicated entry?

Reply:
a bad merge. Fixed in https://github.com/telefonicaid/lightweightm2m-iotagent/pull/124/commits/681f356009d26b65ee35bc94d3753b8028820524